### PR TITLE
Added motion poll pdf (Fixes #1710).

### LINKED ISSUE #1710: Add MotionPoll PDF link

### PR DESCRIPTION
**Pull request by emanuelschuetze**
I've removed poll_number from pdf to make it more consistently with assignmentpoll pdf which also requires poll.id only.

### CODE REVIEW

File: openslides/motions/views.py [hunk 1]
```diff
@@ -333,28 +333,16 @@ class PollPDFView(PDFView):
     required_permission = 'motions.can_manage'
     top_space = 0
```
Comment (normanjaeckel) [inline openslides/motions/views.py]: Remove one empty line.

File: openslides/motions/views.py [hunk 2]
```diff
@@ -333,28 +333,16 @@ class PollPDFView(PDFView):
...
-        Use the motion id and the poll_number from the url kwargs to get the
-        object.
-        """
-        try:
-            obj = self._object
-        except AttributeError:
-            queryset = MotionPoll.objects.filter(
-                motion=self.kwargs['pk'],
-                poll_number=self.kwargs['poll_number'])
-            obj = get_object_or_404(queryset)
```
Comment (normanjaeckel) [inline openslides/motions/views.py]: Remove import of `get_object_or_404`

File: openslides/motions/views.py (hunk 1 above)
Comment (emanuelschuetze) [inline openslides/motions/views.py]: done.

File: openslides/motions/views.py (hunk 2 above)
Comment (emanuelschuetze) [inline openslides/motions/views.py]: done.

**Comment by emanuelschuetze** [comment]
Fixed 2 comments and rebased.

**Pull request merged**